validator-api/src/network_monitor/monitor/sender.rs

```diff
@@ -81,6 +81,8 @@ pub(crate) struct PacketSender {
     // behaviour is unlikely.
     active_gateway_clients: HashMap<[u8; PUBLIC_KEY_LENGTH], GatewayClient>,
 
+    // I guess that will be required later on if credentials are got per gateway
+    // aggregated_verification_key: Arc<VerificationKey>,
```

Comment:
This seems to be part of some other PR

Reply:
To be honest, I'd prefer to leave it there as a note for the next person looking at this code as setting up multiple concurrent gateway clients is super tricky and any suggestion there might be useful (unless you really dislike that, then I will remove that commented line : P )